core/domain/opportunity_services.py

```diff
@@ -498,27 +463,50 @@ def get_translation_opportunities(language_code, topic_name, cursor):
         .ExplorationOpportunitySummaryModel.get_all_translation_opportunities(
             page_size, cursor, language_code, topic_name))
     opportunities = []
-    suggestion_ids = []
-    opportunity_ids = []
+    exp_id_to_in_review_count = (
+        _build_exp_id_to_translation_suggestion_in_review_count(
+            exp_opportunity_summary_models, language_code))
+    for exp_opportunity_summary_model in exp_opportunity_summary_models:
+        exp_opportunity_summary = (
+            get_exploration_opportunity_summary_from_model(
+                exp_opportunity_summary_model))
+        if exp_opportunity_summary.id in exp_id_to_in_review_count:
+            exp_opportunity_summary.translation_in_review_counts = {
+                language_code: exp_id_to_in_review_count[
+                    exp_opportunity_summary.id]
+            }
+        opportunities.append(exp_opportunity_summary)
+    return opportunities, cursor, more
+
+
+def _build_exp_id_to_translation_suggestion_in_review_count(
+        exp_opportunity_summary_models, language_code):
+    """Returns a dict mapping exploration ID to corresponding translation
```

Comment:
Perhaps "... to the corresponding count of translation suggestions that are currently in review." or similar -- I had trouble parsing the last part of the sentence as it currently stands and was wondering what a "review count" was.